Doc: update content (#657)
algolia/docsearch: docs/src/faq.md

could help. Feel free to [send us an email][1] as well, and we'll provide all
the help we can.

## How many records are created by DocSearch?

> First of all, the [parameter `nb_hits`][9] available from your configuration and

Reviewer: ```suggestion
The [property `nb_hits`][9] in your configuration keeps track of the number of records
```

> output after every crawl is the number of records extracted from your website.

Reviewer: ```suggestion
records that were extracted and indexed by the last DocSearch run. It is updated
automatically at each run.
```

> The DocSearch scraper follow [the recommended atomic-reindexing strategy][10].

Reviewer: ```suggestion
The DocSearch scraper follows [the recommended atomic-reindexing strategy][10].
```

It creates a brand new temporary index to populate the data scraped from your
website. Once the crawl is successfully achieved, this temporary index overwites

> the old index defined from your configuration with the key `index_name`.

Reviewer: ```suggestion
the old index defined in your configuration with the key `index_name`.
```
